cdap-distributions/src/packer/scripts/cookbook-setup.sh

```diff
@@ -25,6 +25,9 @@ for cb in cdap hadoop idea maven nodejs openssh; do
   knife cookbook site install $cb || die "Cannot fetch cookbook $cb"
 done
 
+# need nodejs < 3.0.0 until CDAP UI nodeVersion is a version they provide *.xz downloads for
+knife cookbook site install nodejs 2.4.4 || die "Cannot fetch ark cookbook 2.1.0"
```

Comment:
Bad error message, introduced by me in the original PR :( fixed in https://github.com/caskdata/cdap/pull/8522.  We can either backport if appropriate, fix here, or fix later